common/cpp/src/google_smart_card_common/pp_var_utils/extraction.cc

```diff
@@ -18,32 +18,8 @@
 
 #include <google_smart_card_common/numeric_conversions.h>
 
-// Error messages and related bits of strings
-const char kErrorWrongType[] =
```

Comment:
this string is used multiple times below, so I would keep it defined as variable here

Reply:
Done

Reply:
Done (I was afraid that compiler won't be able to verify the printf format specifiers in that case, but looks like it's able to understand the constants defined in the same file as well)